README.md

```diff
@@ -1 +1,185 @@
+  <ul>
+    <li><a href="https://expressjs.com/">Express.js</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+
+### Key Features <a name="key-features"></a>
+
+
+- **[Use Figma Template]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://yourdeployedapplicationlink.com)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+git clone url
+### Prerequisites
+
+In order to run this project you need:
+
+Browser
+Git 
+code editor/Visual Studio Code
+
+### Setup
+
+Clone this repository to your desired folder:
+
+git clone url (https://github.com/Hotdog39duty/Portfolio-setup-and-mobile-first.git)
+
+### Usage
+
+To run the project, execute the following command:
+
+git clone
+cd The portfolio
+code .
+
+### Run tests
+
+To run tests, run the following command:
+
+
+### Deployment
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 👥 Authors <a name="authors"></a>
+
+
+👤 **Author1**
+
+- GitHub: [@githubhandle](https://github.com/Hotdog39duty/githubhandle)
+- Twitter: [@twitterhandle](https://twitter.com/khanya_mvoko/twitterhandle)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/Khanya-mvoko/linkedinhandle)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🔭 Future Features <a name="future-features"></a>
+
+
+- [ ] **[Flexbox]**
+- [ ] **[CSSGrid]**
+- [ ] **[position]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## ⭐️ Show your support <a name="support"></a>
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+> Give credit to everyone who inspired your codebase.
+
+I would like to thank... Microverse as i got a chance to use their template and try remake,
+also thanks to my collaborator and coding partners for helping me so much.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FAQ (optional) -->
+
+## ❓ FAQ (OPTIONAL) <a name="faq"></a>
+
+
+- **[How long does it take to remake this template?]**
+
+  - [Depends on how much effort you put towards the project and the information you understand and putting in making 
+  the website from nothing.]
+
+- **[Could anyone be able to add Javascript?]**
+
+  - [Yes you can and can use for mutiple functions im the program]
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="LICENSE"></a>
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] As suggested by the previous reviewer, kindly your license file. its still not added to the root of your project. doing so makes your work more professional 👍